target/linux/lantiq/xrx200/config-5.4

```diff
@@ -16,7 +16,7 @@ CONFIG_CRYPTO_MANAGER=y
 CONFIG_CRYPTO_MANAGER2=y
 CONFIG_CRYPTO_NULL2=y
 CONFIG_CRYPTO_ZSTD=y
-CONFIG_EXTRA_FIRMWARE="lantiq/xrx200_phy11g_a14.bin lantiq/xrx200_phy11g_a22.bin lantiq/xrx200_phy22f_a14.bin lantiq/xrx200_phy22f_a22.bin"
+CONFIG_EXTRA_FIRMWARE="lantiq/xrx200_phy11g_a14.bin lantiq/xrx200_phy11g_a22.bin lantiq/xrx200_phy22f_a14.bin lantiq/xrx200_phy22f_a22.bin lantiq/xrx300_phy11g_a21.bin lantiq/xrx300_phy22f_a21.bin"
```

Comment:
Adding the GPHY FW here is problematic.
Under which license did you receive this firmware?

The GPHY Firmware is distributed here for example:
https://intel.prpl.dev/intel/ltq_fw_phy_ip 
This the license does not allow a redistribution.

We could for example extract it from the flash partition at runtime and load the firmware after the kernel booted up. This should be legally save.

Reply:
Whole [DWR966 firmware](https://github.com/brunompena/dwr-966) is licensed under GPL.
I don't see anywhere that gphy firmware is under a different license.

It will be easier to convert GPHY firmware to a package like in the dsl firmware and load the switch driver after rootfs mounting. Is this acceptable?

Reply:
With the DSA switch driver it is possible to load the GPHY firmware later from user space.
Should we wait till that gets merged?

Reply:
I am planning to switch to a DSA driver. Previously, DSA driver crashed after a few seconds under iperf. This weekend I will test this driver with the latest patches.